app/gui2/src/components/widgets/NumericInputWidget.vue

```diff
@@ -97,7 +97,7 @@ const inputStyle = computed<StyleValue>(() => {
 
 function blur() {
   inputFieldActive.value = false
-  emit('update:modelValue', currentInputValue.value)
+  emit('update:modelValue', editedValue.value)
 }
 
 /** To prevent other elements from stealing mouse events (which breaks blur),
```

Comment:
I'm not sure if I'm getting what this comment means. What is possible other element stealing mouse click? Why don't we just use useAutoBlur here (and have event handler regardless if input is blurred or not)?

Reply:
Clicking on GraphEditor’s background prevents default handling of the browser event (for handling nodes selection), so we’re not getting `blur` from the browser. `useAutoBlur` is not optimal in this case, because it requires setting up a callback per every input on the screen, and triggering them on every click anywhere in the app. With the current implementation, we set the callback for active input only, so the performance is not dependent on number of input fields. I will extend a comment.